test/api/liskApi.js

```diff
@@ -1,28 +1,24 @@
-if (typeof module !== 'undefined' && module.exports) {
-	var common = require('../common');
-	var lisk = common.lisk;
-	var privateApi = common.privateApi;
-	var utils = common.utils;
-	var sinon = common.sinon;
-	process.env.NODE_ENV = 'test';
-}
+var liskApi = require('../../lib/api/liskApi');
+var privateApi = require('../../lib/api/privateApi');
+var utils = require('../../lib/api/utils');
+var transactionModule = require('../../lib/transactions/transaction');
 
 describe('Lisk.api()', function () {
 
-	var LSK = lisk.api();
+	var LSK = liskApi();
 
-	describe('lisk.api()', function () {
+	describe('liskApi()', function () {
 
-		it('should create a new instance when using lisk.api()', function () {
+		it('should create a new instance when using liskApi()', function () {
 			(LSK).should.be.ok();
 		});
 
-		it('new lisk.api() should be Object', function () {
+		it('new liskApi() should be Object', function () {
 			(LSK).should.be.type('object');
 		});
 
 		it('should use testnet peer for testnet settings', function () {
-			var TESTLSK = lisk.api({ testnet: true });
+			var TESTLSK = liskApi({ testnet: true });
 
 			(TESTLSK.port).should.be.equal(7000);
```

Comment:
this should be defined as a variable, as testnet does not mean 7000 on end user machines